make-options.json

```diff
@@ -3,6 +3,7 @@
         "ANT",
         "Gulp",
         "NuGetInstaller",
+        "PublishSymbols",
```

Comment:
i had issues building the other packages on Windows. I'm not sure if i regressed something or if you have some files that i don't - for now, don't git clean -fdx

i'll sync a prior commit and send the errors i'm hitting in case you know offhand what the issue is.